_studio/mfx_lib/shared/include/mediasdk_version.h

```diff
@@ -18,27 +18,23 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-// This file should be included by exactly one cpp in each MediaSDK plugin
+// This file have specific information for Linux, Android, OSX.
 
 #pragma once
 
-/* These string constants set Media SDK version information for Linux, Android, OSX. */
-#ifndef MFX_PLUGIN_FILE_VERSION
-#define MFX_PLUGIN_FILE_VERSION "0.0.0.0"
-#endif
-#ifndef MFX_PLUGIN_PRODUCT_NAME
-#define MFX_PLUGIN_PRODUCT_NAME "Intel(R) Media SDK Plugin"
-#endif
-#ifndef MFX_PLUGIN_PRODUCT_VERSION
-#define MFX_PLUGIN_PRODUCT_VERSION "0.0.000.0000"
-#endif
+#ifdef __linux__
+#include "va/va.h"
+
 #ifndef MFX_FILE_DESCRIPTION
 #define MFX_FILE_DESCRIPTION "File info ought to be here"
 #endif
 
-const char* g_MfxProductName = "mediasdk_product_name: " MFX_PLUGIN_PRODUCT_NAME;
-const char* g_MfxCopyright = "mediasdk_copyright: Copyright(c) 2007-2018 Intel Corporation";
-const char* g_MfxProductVersion = "mediasdk_product_version: " MFX_PLUGIN_PRODUCT_VERSION;
-const char* g_MfxFileVersion = "mediasdk_file_version: " MFX_PLUGIN_FILE_VERSION;
-const char* g_MfxFileDescription = "mediasdk_file_description: " MFX_FILE_DESCRIPTION;
+const char* g_MfxProductName     = "mediasdk_product_name: Intel(R) Media SDK";
+const char* g_MfxCopyright       = "mediasdk_copyright: Copyright(c) 2007-2019 Intel Corporation";
+const char* g_MfxApiVersion      = "mediasdk_api_version: " MFX_API_VERSION;
+const char* g_MfxVersion         = "mediasdk_version: master";
```

Comment:
emmm. I have some feeling that we should have not only master here. From time to time at least:).

Reply:
I believe it supposed to be corrected manually after branch out.